gxformat2/model.py

```diff
@@ -1,7 +1,136 @@
 """Abstractions for dealing with Format2 data."""
-from typing import cast, Dict, List, Union
+import logging
+import os
+from typing import (
+    Any,
+    cast,
+    Dict,
+    List,
+    Optional,
+    Union,
+)
+
+from typing_extensions import TypedDict
+
+log = logging.getLogger(__name__)
 
 DictOrList = Union[Dict, List]
+ConnectDict = dict
+
+
+EmbeddedLink = TypedDict("EmbeddedLink", {"$link": str})
```

Comment:
@jmchilton This `EmbeddedLink` type seems unused and generates this warning when building the docs:

```
WARNING: invalid signature for autoattribute ('gxformat2.model::EmbeddedLink.$link') [autodoc]
WARNING: don't know which module to import for autodocumenting 'gxformat2.model::EmbeddedLink.$link' (try placing a "module" or "currentmodule" directive in the document, or giving an explicit module name) [autodoc]
```

Can we drop it?